### DISCUSSION TITLE
Add a tool that divides by trip the loot tracker plugin

### DISCUSSION THREAD

**Discussion by Pogonias** [runelite/runelite#15807] [Ideas]
Adding a way to group loot by trip would be really good for a lot of activity like revenants where a lot of people leave to bank at a certain loot value or just split two grind sessions where two setups are experienced and can compare the number of monsters kill without having to reset the plugin entierly.

I think just a button that divide groups manualy coulds be a fine and easy addition for loot tracker plugin but maybe a automatic way that track when you teleport out could work but it might be to much. Just throwing the idea. ¯\_(ツ)_/¯

**Comment by Daggerdoc** [comment]
I think a similar idea that could also work really well is to have the ability to track loot based on like a session. 

Lets say I killed 100 blue dragons 10 days ago and I want to kill 100 more today. Right now I'd end up with 200 different entries for 1 blue dragon each or 1 entry for 200 blue dragons. I think it'd be cool if you could end up with two entries of 100 dragons each. 

You could use the time between kills to determine whether to make a new entry or not. Like if it's been an hour since I killed a blue dragon make a new blue dragon entry and reset that timer. The time it takes to define a new session would also be a variable the user could set themselves.

**Reply by DapperMickie** [reply]
I think a "start new session" button would be the best way to implement this, I feel like time based isn't as useful.

**Reply by DominickCobb-rs** [reply]
Loot Logger plugin has a "current session data".

**Reply by Daggerdoc** [reply]
wait really? where is it? I've been looking for something like that forever!
